extensions/wikia/WikiaMaps/controllers/WikiaMapsSpecialController.class.php

```diff
@@ -45,6 +45,9 @@ public function index() {
 		$this->wg->SuppressPageHeader = true;
 		$this->wg->out->setHTMLTitle( wfMessage( 'wikia-interactive-maps-title' )->escaped() );
 
+		// SUS-936: Don't load article related assets here
+		$this->wg->out->setArticleRelated( false );
+
 		if ( is_numeric( $this->getPar() ) ) {
 			if ( $this->getRequest()->getVal( '_escaped_fragment_' ) === null ) {
 				$this->forward( 'WikiaMapsSpecial', 'map' );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "WikiaMapsSpecial" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)